src/gateways/documents-api.test.ts

```diff
@@ -44,4 +44,28 @@ describe('Documents api gateway', () => {
       });
     });
   });
+
+  describe('GET request to /claims/{claimId}/download_links', () => {
+    const claimId = '123-abc-claimId';
+    const expectedData = 'www.downloadlinkfordocument.com';
+    const expectedStatus = 200;
+    const expectedResponse = { data: expectedData, status: expectedStatus };
+
+    beforeEach(() => {
+      client.get.mockResolvedValue({ data: expectedResponse });
+    });
+
+    it('the request returns the correct response', async () => {
+      const response = await gateway.getDocumentPreSignedUrl(claimId);
+      expect(response).toEqual(expectedResponse);
+      expect(client.get).toHaveBeenCalledWith(
+        `/api/v1/claims/${claimId}/download_links`,
+        {
+          headers: {
+            Authorization: process.env.DOCUMENTS_API_GET_DOCUMENTS_TOKEN,
+          },
+        }
+      );
+    });
+  });
```

Comment:
We could also have a test for the error case

Reply:
I've added a new test to check the internal server error. It differs from the one that exists in the gateway elsewhere due to the test not actually catching the error in the assertion. Found out that I was getting a `UnhandledPromiseRejectionWarning` and not an outright fail, so I did some research and found that the test should be structured differently. 

https://stackoverflow.com/questions/56291310/what-is-the-meaning-of-unhandledpromiserejectionwarning-in-jest-tests

The `expect.assertions(1)` is a way to check if async calls were actually made. https://jestjs.io/docs/expect#expectassertionsnumber